src/Screen/Screen.php

```diff
@@ -29,6 +29,12 @@ abstract class Screen extends Controller
      * Example: dashboard/my-screen/{method?}
      */
     private const COUNT_ROUTE_VARIABLES = 1;
+    
+    /**
+     * The view that should be rendered
+     *
+     */
+    protected $view = 'platform::layouts.base';
```

Comment:
Let's use a different method to get the template because a property with such a famous name might already be taken. For example method `$this->screenBaseView()`